.cbmc-batch/include/proof_helpers/aws_byte_cursor_read_common.h

```diff
@@ -20,14 +20,14 @@
 void aws_byte_cursor_read_common_harness() {
     /* parameters */
     struct aws_byte_cursor cur;
-    size_t length;
-    DEST_TYPE *dest = can_fail_malloc(length);
+    DEST_TYPE *dest = can_fail_malloc(sizeof(*dest));
 
     /* assumptions */
     ensure_byte_cursor_has_allocated_buffer_member(&cur);
     __CPROVER_assume(aws_byte_cursor_is_valid(&cur));
+    __CPROVER_assume(cur.len >= BYTE_WIDTH);
     __CPROVER_assume(AWS_MEM_IS_READABLE(cur.ptr, BYTE_WIDTH));
-    __CPROVER_assume(AWS_OBJECT_PTR_IS_WRITABLE(dest));
+    __CPROVER_assume(dest != NULL);
```

Comment:
Instead of just replacing it, should we add also a comment? So we can put it back once the issue is fixed on CBMC?

Reply:
Actually, now with the updated `AWS_OBJECT_PTR_IS_WRITABLE`, do we need to remove it from here?